### PR TITLE
Add Array Datatype to UCR

### PR DESCRIPTION
**Pull request by sheelio**
Added a new array data type.  It will just return the input if its already an array.  If not, will take the input and return a single element array.

### CODE REVIEW

File: corehq/apps/userreports/expressions/getters.py [hunk 1]
```diff
@@ -133,6 +133,15 @@ def transform_unicode(item):
         return None
 
 
+def transform_array(item):
+    if isinstance(item, list):
+        return item
+    try
```
Comment (dimagimon) [inline corehq/apps/userreports/expressions/getters.py]: E901 SyntaxError: invalid syntax

File: corehq/apps/userreports/expressions/getters.py [hunk 2]
```diff
@@ -133,6 +133,15 @@ def transform_unicode(item):
...
+        return [item]
```
Comment (dimagimon) [inline corehq/apps/userreports/expressions/getters.py]: E113 unexpected indentation

File: corehq/apps/userreports/expressions/getters.py [hunk 3]
```diff
@@ -133,6 +133,15 @@ def transform_unicode(item):
...
+    except (InvalidOperation)
+        return None
+
+
```
Comment (dimagimon) [inline corehq/apps/userreports/expressions/getters.py]: W293 blank line contains whitespace

File: corehq/apps/userreports/expressions/getters.py [hunk 4]
```diff
@@ -133,6 +133,15 @@ def transform_unicode(item):
         return None
 
 
+def transform_array(item):
+    if isinstance(item, list):
+        return item
+    try
+        return [item]
+    except (InvalidOperation)
+        return None
```
Comment (dimagimon) [inline corehq/apps/userreports/expressions/getters.py]: E113 unexpected indentation

File: corehq/apps/userreports/expressions/getters.py [hunk 5]
```diff
@@ -133,6 +133,15 @@ def transform_unicode(item):
         return None
 
 
+def transform_array(item):
+    if isinstance(item, list):
+        return item
+    try:
+        return [item]
+    except (InvalidOperation)
```
Comment (dimagimon) [inline corehq/apps/userreports/expressions/getters.py]: E901 SyntaxError: invalid syntax

File: corehq/apps/userreports/expressions/getters.py [hunk 6]
```diff
@@ -133,6 +133,15 @@ def transform_unicode(item):
...
+        return None
```
Comment (dimagimon) [inline corehq/apps/userreports/expressions/getters.py]: E113 unexpected indentation

**Comment by sheelio** [comment]
@NoahCarnahan @nickpell 

For review and then you can merge.  Waiting for tests to pass.

File: corehq/apps/userreports/expressions/getters.py [hunk 7]
```diff
@@ -133,6 +133,15 @@ def transform_unicode(item):
         return None
 
 
+def transform_array(item):
+    if isinstance(item, list):
+        return item
+    try:
+        return [item]
+    except (InvalidOperation):
```
Comment (nickpell) [inline corehq/apps/userreports/expressions/getters.py]: We can remove this except clause

**Comment by nickpell** [comment]
@sheelio you should write code more often :)

**Comment by snopoke** [comment]
looks good

**Comment by czue** [comment]
:+1: thanks!

**Pull request merged**